packages/mwp-core/src/util/linkify.js

```diff
@@ -31,9 +53,14 @@ const createLink = (options: Object) => (href: string): string => {
  * @param {Object} options optional modifiers
  * @return {String} The modified text.
  */
-export default function linkify(text: string, options?: Object = {}): string {
+export default function linkify(
+	text: string,
+	options?: Object = {},
+	isDomainLogicEnabled?: boolean,
```

Comment:
Do we need this separate flag? How about just validating that `domains` is defined?